tests/manual-test-cases/Group19-VIC-Product/19-5-Configuration-1-VC6.5-ESX6.5-VIC-OVA-Harbor-Insecured.md

```diff
@@ -0,0 +1,99 @@
+Test 19-5 - Configuration 1 VC6.5 ESX6.5 VIC OVA Insecured
+=======
+
+#Purpose:
+To verify the VIC Product (Engine/Harbor) work using insecure registry with a VC6.5 and ESX6.5 server
+
+#References:
+[1 - VIC+Harbor Integration Test Plan](https://confluence.eng.vmware.com/pages/viewpage.action?spaceKey=corevc&title=VIC+-+Harbor+Integration+Test+Plan)
+
+#Environment:
+* This test requires access to VMWare Nimbus cluster for dynamic ESXi and vCenter creation
+* Login to Harbor using an LDAP user1 (say admin role)
```

Comment:
I would name the user `admin` if admin privileges are required to perform the actions done by this user